packaging/installer/kickstart.sh

```diff
@@ -745,7 +745,7 @@ confirm() {
 update() {
   updater="${ndprefix}/usr/libexec/netdata/netdata-updater.sh"
 
-  if [ -x "${updater}" ]; then
+  if run_as_root [ -x "${updater}" ]; then
```

Comment:
Could we change these to use `test` instead of `[`?

I feel like that will be more readable here.